exercises/Functions-6-R.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: exercise
+title: Functions 6
+language: R
+---
+
+This is a follow up to [Strings 6]({{ site.baseurl }}/exercises/Strings-6/).
+
+A colleague has produced a file with one DNA sequence on each line. Download
+[the file]({{ site.baseurl }}/data/dna_sequences_1.txt) and load it into R using
+`read.csv()`. The file has no header and is separated by white space (`’’`).
```

Comment:
I'm not sure what this means. Each sequence is one a single line. What is the "separated by whitespace"?